### PR TITLE
Fix cursor should not be pointer when record image identifier is not Editable

### PR DESCRIPTION
**Pull request by sachinks07**
- This PR solves the issue Cursor should not be "pointer" when record image identifier is not editable #7277

### CODE REVIEW

File: packages/twenty-front/src/modules/object-record/record-show/components/RecordShowContainer.tsx [hunk 1]
```diff
@@ -201,6 +201,7 @@ export const RecordShowContainer = ({
         </FieldContext.Provider>
       }
       avatarType={recordIdentifier?.avatarType ?? 'rounded'}
+      isAvatarEditable={objectNameSingular === 'person'}
```
Comment (martmull) [inline packages/twenty-front/src/modules/object-record/record-show/components/RecordShowContainer.tsx:204]:
```suggestion
```

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ type ShowPageSummaryCardProps = {
   title: ReactNode;
   loading: boolean;
   isMobile?: boolean;
+  isAvatarEditable?: boolean;
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx:26]:
```suggestion
```

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx [hunk 3]
```diff
@@ -107,6 +109,7 @@ export const ShowPageSummaryCard = ({
   title,
   loading,
   isMobile = false,
+  isAvatarEditable = false,
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx:112]:
```suggestion
```

File: packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx [hunk 4]
```diff
@@ -130,7 +133,7 @@ export const ShowPageSummaryCard = ({
 
   return (
     <StyledShowPageSummaryCard isMobile={isMobile}>
-      <StyledAvatarWrapper>
+      <StyledAvatarWrapper isAvatarEditable={isAvatarEditable}>
```
Comment (martmull) [inline packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx:136]:
```suggestion
      <StyledAvatarWrapper isAvatarEditable={!!onUploadPicture}>
```

**Review by martmull — CHANGES_REQUESTED** [review]
Thanks, added a suggestion to simplify changes

**Comment by sachinks07** [comment]
@martmull Thanks for the suggestion. Have made the changes with respect to the suggestions.

**Review by martmull — APPROVED** [review]
LGTM thank you!

**Pull request merged**